### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by mariocraft987** [cytrinc/bark-coding#106] [General]
>I tried out the link given on your profile, Bark is so cool! Keep up the good work :D - -ExoByte-

>someone make a copy of scratch called "bark" and make this the "bark dog"." - Psychotic_Jelly_Bean
>no! thats just- like weird/uneccesary" - KittyKatt64

These are real feedback for Bark Coding

**Comment by raynepaws** [comment]
that is actually so awesome
where'd you get these from?

**Reply by mariocraft987** [reply]
1 from a alt account then this comment: https://scratch.mit.edu/projects/407181845/#comments-396137259
